Source/cielim/Private/RenderingFunctionsLibrary.h

```diff
@@ -22,21 +22,12 @@ class  CIELIM_API URenderingFunctionsLibrary :
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
 	static void ApplyPSF_Gaussian();
 
-	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
-	static void WriteImage();
-
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
 	static void ApplyCosmicRays();
 
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
-	static void ApplyReadNoise();
-
-	static void ReadImage(){};
-
-	
-
-	
-};
-
-
+	static FString ApplyReadNoise(FString Filepath, float ReadNoiseSigma, float SystemGain);
 
+	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
+	static FString ApplySignalGain(FString Filepath, float ImageGain, float DesiredGain);
+};
```

Comment:
:small_red_triangle_down: New line at end of file

Reply:
New line -> added